src/sessionManager.ts

```diff
@@ -45,9 +45,33 @@ export default function SessionManager(
                 self.endSession();
                 self.startNewSession();
             } else {
+                const user = mpInstance.Identity.getCurrentUser()
+                let needToIdentify = false
+
+                if (user) {
+                    // If userIdentities is an empty object but identifyRequest
+                    // has at least a customerid, we will force the identify()
+                    // call, regardless of whether any user data is already
+                    // persisted.
+                    const storedUserIdentities = user
+                        .getUserIdentities()?.userIdentities
+                    const hasStoredCustomerId =
+                        storedUserIdentities != null &&
+                        typeof storedUserIdentities.customerid === "string"
```

Comment:
Is the purpose of checking if the customerid is as string to verify that it's present? I wonder if checking `typeof storedUserIdentities.customerid !== "undefined"` is more obvious?

We have a utility method called `isEmpty` that might work, but it was intended for objects/arrays. 

If you need to make sure it's a string, we have an `isString` utility method as well.